src/coreComponents/constitutive/fluid/CellBasedFluxSlurryFluidAccessors.hpp

```diff
@@ -0,0 +1,57 @@
+/*	
+ * ------------------------------------------------------------------------------------------------------------	
+ * SPDX-License-Identifier: LGPL-2.1-only	
+ *	
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC	
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University	
+ * Copyright (c) 2018-2020 Total, S.A	
+ * Copyright (c) 2020-     GEOSX Contributors	
+ * All right reserved	
+ *	
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.	
+ * ------------------------------------------------------------------------------------------------------------	
+ */
+
+#ifndef GEOSX_CELLBASEDFLUXSLURRYFLUIDACCESSORS_HPP
+#define GEOSX_CELLBASEDFLUXSLURRYFLUIDACCESSORS_HPP
+
+#include "SlurryFluidExtrinsicData.hpp"
+
+#include "physicsSolvers/fluidFlow/StencilAccessors.hpp"
```

Comment:
Is this tool really ment to live only for flow or is it broader than that?

Reply:
The stencil accessors are used in the kernels that need to access non-local data (i.e., when you need to access data in the two cells adjacent to a face). For now, this pattern is only present in the flow part of the code for the assembly of the fluxes and the computation of flux-like quantities (aquifer, CFL) and it may stay like that for a while. So, I think it makes sense to keep these accessors in `physicsSolvers/fluidFlow`. We can always move them later if/when we have another use case.